src/pages/EnablePayments/VerifyIdentity/VerifyIdentity.tsx

```diff
@@ -0,0 +1,138 @@
+import * as Wallet from '@userActions/Wallet';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {WalletOnfido} from '@src/types/onyx';
+
+const DEFAULT_WALLET_ONFIDO_DATA = {
+    isLoading: false,
+    hasAcceptedPrivacyPolicy: false,
+    sdkToken: '',
+    applicantID: '',
+};
+
+type VerifyIdentityOnyxProps = {
+    /** The wallet onfido data */
+    walletOnfidoData: OnyxEntry<WalletOnfido>;
+};
+
+type VerifyIdentityProps = VerifyIdentityOnyxProps;
+
+const ONFIDO_ERROR_DISPLAY_DURATION = 10000;
+
+function VerifyIdentity({walletOnfidoData = DEFAULT_WALLET_ONFIDO_DATA}: VerifyIdentityProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const openOnfidoFlow = () => {
+        BankAccounts.openOnfidoFlow();
+    };
+
+    const handleOnfidoSuccess = useCallback(
+        (onfidoData: OnfidoData) => {
+            BankAccounts.verifyIdentity({
+                onfidoData: JSON.stringify({
+                    ...onfidoData,
+                    applicantID: walletOnfidoData?.applicantID,
+                }),
+            });
+            BankAccounts.updateAddPersonalBankAccountDraft({isOnfidoSetupComplete: true});
+        },
+        [walletOnfidoData?.applicantID],
+    );
+
+    const onfidoError = ErrorUtils.getLatestErrorMessage(walletOnfidoData) ?? '';
+
+    const handleOnfidoError = () => {
+        Growl.error(translate('onfidoStep.genericError'), ONFIDO_ERROR_DISPLAY_DURATION);
+    };
+
+    const goBack = () => {
+        Wallet.updateCurrentStep(CONST.WALLET.STEP.ADDITIONAL_DETAILS);
+    };
+
+    return (
+        <ScreenWrapper testID={VerifyIdentity.displayName}>
+            <HeaderWithBackButton
+                title={translate('onfidoStep.verifyIdentity')}
+                onBackButtonPress={goBack}
+            />
+            <View style={[styles.ph5, styles.mt3, {height: CONST.BANK_ACCOUNT.STEPS_HEADER_HEIGHT}]}>
+                <InteractiveStepSubHeader
+                    startStepIndex={2}
+                    stepNames={CONST.WALLET.STEP_NAMES}
+                />
+            </View>
+            <FullPageOfflineBlockingView>
+                <ScrollView contentContainerStyle={styles.flex1}>
+                    {walletOnfidoData?.hasAcceptedPrivacyPolicy ? (
+                        <Onfido
+                            sdkToken={walletOnfidoData?.sdkToken ?? ''}
+                            onUserExit={goBack}
+                            onError={handleOnfidoError}
+                            onSuccess={handleOnfidoSuccess}
+                        />
+                    ) : (
+                        <>
+                            <View style={[styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter, styles.m5, styles.ph5]}>
+                                <Icon
+                                    src={Illustrations.ToddBehindCloud}
+                                    fill="black"
```

Comment:
@koko57  why do we add fill="black" here

https://github.com/Expensify/App/assets/141406735/3d9cb73b-1a1d-4bee-8fe6-358043bbbb60

Reply:
@DylanDylann fixed!

Reply:
@mountiny @DylanDylann all comments addressed